pkg/serverconf/murmurcompat.go

```diff
@@ -0,0 +1,58 @@
+package serverconf
+
+import (
+	"log"
+)
+
+var murmurCompatRules = map[string]string{
```

Comment:
like mentioned above, drop this (file); use common names with murmur

Reply:
See above.